src/main/kotlin/de/randombyte/unity/Unity.kt

```diff
@@ -156,7 +153,7 @@ class Unity @Inject constructor(
 
         Sponge.getCommandManager().register(this, CommandSpec.builder()
                 // the root unity/marry command is the 'request unity'-command
-                .permission(PLAYER_PERMISSION)
```

Comment:
To keep backwards compatibility, please keep the `PLAYER_PERMISSION` and do it like so

```
.permission("$PLAYER_PERMISSION.marry")
```

It is just to keep it compatible with all the permission setups out there.